### DISCUSSION TITLE
What are the risks to installing under admin priviledges?

### DISCUSSION THREAD

**Discussion by midrare** [scoopinstaller/install#88] [Q&A]
The install guide says:

> Installation under the administrator console has been disabled by default for **security considerations**.

What are these security considerations? What are the differences when installing under admin vs normal user?

**Comment by akehsanz** [comment]
Yes.. i was wondering the SAME. What are the security considerations to take on notice when installing under admin ?? Pls reply anyone with ample knowledge..
